BaseClasses.py

```diff
@@ -767,8 +767,23 @@ def item_count(self, item: str, player: int) -> int:
         Utils.deprecate("Use count instead.")
         return self.count(item, player)
 
+    def has_from_list(self, items: Iterable[str], player: int, count: int) -> bool:
```

Comment:
to be honest, the doc string do make it clear, but when i see `state.has_from_list(["itemA", "ItemB"], 3, player)` it would be unclear to me at first glance what it does
`has_from_list` to me from its method name looks very simular to `state.has_any(["itemA", "ItemB"], player)`
so my recommendation would be to name it `has_atleast()`